app/src/main/res/layout/context_menu_title.xml

```diff
@@ -1,15 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <TextView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:maxLines="3"
-    android:gravity="center_vertical"
     android:ellipsize="end"
+    android:fontFamily="@string/font_roboto_regular"
+    android:gravity="center_vertical"
```

Comment:
nit: please set these lines to their original order! Reordering lines makes it more difficult to review your code and follow git history! You should have a really good reason for when you reorder lines!

Reply:
Oops autoformatter again.